coverage/clover.xml

```diff
@@ -1,11 +1,44 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<coverage generated="1630686735163" clover="3.2.0">
-  <project timestamp="1630686735163" name="All files">
-    <metrics statements="2" coveredstatements="2" conditionals="0" coveredconditionals="0" methods="1" coveredmethods="1" elements="3" coveredelements="3" complexity="0" loc="2" ncloc="2" packages="1" files="1" classes="1"/>
-    <file name="sum.js" path="/Users/hackreactor/work/blueOcean/Fitter/sum.js">
+<coverage generated="1631251757614" clover="3.2.0">
```

Comment:
Haven't seen this kind of file before, curious as to what this is for?

Reply:
I believe this part of the test coverage which I'm not sure if we should be pushing up to the repo.